packages/git.rb

```diff
@@ -3,33 +3,35 @@
 class Git < Package
   description 'Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.'
   homepage 'https://git-scm.com/'
-  @_ver = '2.31.1'
-  version "#{@_ver}-1"
+  @_ver = '2.32.0'
```

Comment:
Should we remove the git update from #5837?

Reply:
It's literally the same package file in both PRs. And they're static. So no change is needed.